src/ontology/uberon-edit.obo

```diff
@@ -224009,6 +224009,19 @@ is_a: UBERON:8600006 ! visceral striated muscle tissue
 relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
 property_value: dcterms-date "2023-02-21T09:26:41Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600009
+name: subsegmental bronchus
+def: "Branch of conducting airway between a segmental bronchus and a preterminal bronchiole." [ISBN:0781785898, ISBN:9780702077050, Wikipedia:Bronchus]
+comment: Like other bronchi, the walls of subsegmental bronchus contain plates of cartilage. Notably, the cartilage plates become smaller and more widely dispersed with each branching. {xref="https://doi.org/10.1016/B978-0-12-381361-9.00009-3", xref="PMID:14913504"}
```

Comment:
The doi link is behind a paywall, but the following picture is in the chapter referenced. Interestingly, here they refer the subsegmental bronchus as intrasegmental bronchus, and therefore it will be added as a related synonym.
![image](https://user-images.githubusercontent.com/94959119/230327625-17e86543-082a-45fa-8c90-16070caded62.png)

In the PMID reference, the text is more general to all bronchi, but it reflects that the most distally the bronchi are, the smaller and widely dispersed the cartilage plates get.
![image](https://user-images.githubusercontent.com/94959119/230328392-d90c395d-a799-4c03-a658-274a1ae80ae9.png)

Reply:
Thank you for the illustration.
Based on this, `'distally connected to' some 'lobular bronchiole'` should be `'distally connected to' some 'bronchiole'`.